client/rpc-api/src/child_state/mod.rs

```diff
@@ -34,6 +34,7 @@ pub trait ChildStateApi<Hash> {
 	/// RPC Metadata
 	type Metadata;
 
+	/// DEPRECATED: Please use `childstate_getKeysPaged` with proper paging support.
```

Comment:
Do we not actually mark this as deprecated somewhere?

Reply:
I am not really sure. This is just text that will show in rust doc, I copy pasted it from its top storage counterpart.
Maybe there is a better way or could be interesting to remove in a next version with other impacting rpc changes
(I thing the comment for the `state` function was here for rather long).